### DISCUSSION TITLE
Push on github website

### DISCUSSION THREAD

**Discussion by HaoSun1** [community/community#21833] [New to GitHub]
<p>New here, if I have cloned a GitHub repository, how do I push changes to the main branch?  </p>
<p>I only see the pull button where are the push and sync?  </p>
<p>I would really like to avoid coding for now.</p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Hey! If this is your first time using GitHub to program, I suggest heading over to <a href="https://lab.github.com" rel="nofollow noopener">https://lab.github.com</a> for some step by step tutorials on GitHub.  Please let me know if you have questions after running through those tutorials. </p>
